AcknowList.podspec.json

```diff
@@ -15,10 +15,10 @@
     "git": "https://github.com/vtourraine/AcknowList.git",
     "tag": "1.9.5"
   },
-  "source_files": "Source/*.swift",
-  "resources": "Resources/AcknowList.bundle",
+  "source_files": "Sources/*.swift",
+  "resources": "Sources/Resources/AcknowList.bundle",
```

Comment:
Moving the “resources” folder inside of “sources” seems counterintuitive. Is this required for SPM?